libs/package.json

```diff
@@ -2,15 +2,13 @@
   "name": "@audius/sdk",
   "version": "0.0.22",
   "description": "",
-  "main": "dist/index.js",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
   "types": "dist/types.d.ts",
-  "browser": "dist/browser.js",
-  "sdkBrowserDistFile": "dist/sdk.js",
-  "sdkBrowserDistFileMini": "dist/sdk.min.js",
-  "legacy": "dist/legacy.js",
```

Comment:
gotcha so basically removing here and just going with rollup config specifying?

Reply:
Yup, felt weird to have unsupported keys in the package.json just to specify a path in one place